src/components/read/TopBar.js

```diff
@@ -200,7 +204,30 @@ const TopBar = (props) => {
       ""
     );
   };
-
+  const ISLAudioBible = () => {
+    return (
+      <>
+        <Link to="/audiobible">
+          {mobileView ? (
+            <i className={`material-icons ${classes.islIcon}`}>queue_music</i>
+          ) : (
+            <Button
+              variant="outlined"
+              size="small"
+              className={classes.button}
+              title="Sign Language Audio Bible"
+              aria-label="sign language audio bible"
+              target="_blank"
+              rel="noopener"
+              startIcon={<i className="material-icons">queue_music</i>}
+            >
+              ISL Audio
```

Comment:
This can be called Audio
```suggestion
              Audio
```

Reply:
resolved